HISTORY.rst

```diff
@@ -26,6 +26,11 @@ Unreleased Changes (master)
   might be included with the source distribution.
 * Added ``set_tol_to_zero`` to ``convolve_2d`` to allow for in-function masking
   of near-zero results to be set to 0.0.
+* Renamed all parameters involving Spatial Projectsions to the form
```

Comment:
typo: Projectsions -> Projections

Reply:
patched in 5eedf44d06fe3765303952cad6b0a2db435ab5c0